src/oasis-nexus/api.ts

```diff
@@ -752,7 +752,9 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
                               event.body.amount.Amount,
                               paraTimesConfig[runtime].decimals,
                             ),
-                            Denomination: getTokensForScope({ network, layer: runtime })[0].ticker, // TODO find this out from event data
+                            Denomination:
+                              event.body?.Denomination ??
```

Comment:
This doesn't exist. `event.body?.amount.Denomination` exists.
But `event.body.amount.Denomination === ''` above means that it is empty

Reply:
I have seen both existing on different data, IIRC.

Reply:
Well, maybe not.

Reply:
Right, when this code is executed, it's always empty. Removed useless condition in #1285.